contracts/FlowIDTableStaking.cdc

```diff
@@ -622,64 +593,68 @@ pub contract FlowIDTableStaking {
         }
     }
 
-    /// Admin resource that has the ability to remove insufficiently staked nodes
-    /// at the end of the staking auction, pay rewards to nodes at the end of an epoch, and move tokens between buckets
     pub resource Admin {
 
         /// Remove a node from the record
         pub fun removeNode(_ nodeID: String): @NodeRecord {
             let node <- FlowIDTableStaking.nodes.remove(key: nodeID)
                 ?? panic("Could not find a node with the specified ID")
 
-            FlowIDTableStaking.networkingAddressClaimed.remove(key: node.networkingAddress)
-            FlowIDTableStaking.networkingKeyClaimed.remove(key: node.networkingKey)
-            FlowIDTableStaking.stakingKeyClaimed.remove(key: node.stakingKey)
+            FlowIDTableStaking.updateClaimed(path: /storage/networkingAddressesClaimed, node.networkingAddress, claimed: false)
+            FlowIDTableStaking.updateClaimed(path: /storage/networkingKeysClaimed, node.networkingKey, claimed: false)
+            FlowIDTableStaking.updateClaimed(path: /storage/stakingKeysClaimed, node.stakingKey, claimed: false)
 
             return <-node
         }
 
         pub fun startStakingAuction() {
             pre {
-                !FlowIDTableStaking.stakingEnabled: "Cannot start staking auction if it is already in progress"
+                !FlowIDTableStaking.stakingEnabled(): "Cannot start staking auction if it is already in progress"
             }
-            FlowIDTableStaking.stakingEnabled = true
+            let stakingEnabled = FlowIDTableStaking.account.load<Bool>(from: /storage/stakingEnabled)
+
+            FlowIDTableStaking.account.save(true, to: /storage/stakingEnabled)
         }
 
-        /// Iterates through all the registered nodes and if it finds
-        /// a node that has insufficient tokens committed for the next epoch
-        /// it moves their committed tokens to their unstaked bucket
-        ///
-        /// Parameter: approvedNodeIDs: A list of nodeIDs that have been approved
-        /// by the protocol to be a staker for the next epoch.
+        /// Ends the staking Auction by removing any unapproved nodes
+        /// and setting stakingEnabled to false
         pub fun endStakingAuction(approvedNodeIDs: {String: Bool}) {
             pre {
-                FlowIDTableStaking.stakingEnabled: "Cannot end the staking auction if it is not currently in progress"
+                FlowIDTableStaking.stakingEnabled(): "Cannot end the staking auction if it is not currently in progress"
             }
 
+            self.removeUnapprovedNodes(approvedNodeIDs: approvedNodeIDs)
+
+            let stakingEnabled = FlowIDTableStaking.account.load<Bool>(from: /storage/stakingEnabled)
```

Comment:
seems like we don't use this value after loading it

Reply:
yeah, I removed the variable since we don't use it